access-common/src/main/java/edu/unc/lib/dl/ui/service/LorisContentService.java

```diff
@@ -202,16 +204,16 @@ public String getManifest(HttpServletRequest request, List<BriefObjectMetadata>
             manifest.addLabel(label);
         }
 
-        setMedataField(manifest, "Creators", creators);
-        setMedataField(manifest, "Keywords", keywords);
-        setMedataField(manifest, "Subjects", subjects);
-        setMedataField(manifest, "Languages", language);
+        setMetadataField(manifest, "Creators", creators);
+        setMetadataField(manifest, "Keywords", keywords);
+        setMetadataField(manifest, "Subjects", subjects);
+        setMetadataField(manifest, "Languages", language);
 
         Sequence seq = new Sequence(URIUtil.join(manifestBase, "sequence", "normal"));
 
-        List<String> uuidList = new ArrayList<String>();
+        List<String> uuidList = new ArrayList<>();
```

Comment:
Doesn't matter much for data at this scale, but a Set/HashSet is often a good choice for a data structure for quickly checking to see if a value has already occurred.